app.organicmaps.desktop.yml

```diff
@@ -0,0 +1,39 @@
+app-id: app.organicmaps.desktop
```

Comment:
`.desktop` is a bad idea. Really. It should be something like `app.organigcmaps.OrganicMaps`.

Reply:
Why is it a bad idea?

@biodranik would switching [back](https://github.com/organicmaps/organicmaps/pull/2206#discussion_r838309052) to `app.organigcmaps.OrganicMaps` be ok?

Reply:
because
1. .desktop confuse with desktop files. You end up with `something.something.desktop.desktop` as a desktop file name.
2. it is not an app name. The original issue you linked had it right.

(any example of this on flathub is wrong, but hard to fix)

Reply:
I don't understand the problem with .desktop, but if it's critical, `app.organicmaps.flathub` is also ok.

Reply:
it is not supposed to be flatpak / flathub specific. It's actually supposed to be the same if it's a deb or a rpm as this is the same ID used elsewhere... (like to register a dbus session bus, but I haven't figured out if you do or not, as it is optional with Qt).

**The last component of the app-id is supposed to be the name of the application** in some form.

See https://docs.flatpak.org/en/latest/conventions.html#application-ids

`app.organicmaps.OrganicMaps` is IMHO the best choice, the last part being consistent with your .desktop file. BTW .desktop files also use the app-id in their name, but that's something that gets fixed with the `rename-desktop-file`.

Reply:
Then `app.organicmaps` or `app.organicmaps.linux` should be ok.

Reply:
> Then `app.organicmaps` or `app.organicmaps.linux` should be ok.

Neither. The first one is too short, there shall be at least 3 components. The second, well, same as previously. "linux" isn't the name of the application.

Reply:
maybe we could find a compromise? eg. app.organicmaps.OrganicMapsDesktop both satisfies flathub requirements (domain.AppName) and has "desktop" in the name

Reply:
It fits the requirements, so yes.

Reply:
There are applications with `.desktop` suffix though, so it's not like the end of the world. Some developers tend to capitalize the last part, so it would be `app.organicmaps.Desktop` in this case.

Reply:
Funny thing the app now can run on mobiles too, so the `.desktop` makes it really confusing IMO:
https://github.com/flathub/app.organicmaps.desktop/issues/39